src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DefaultLSPDocumentMappingProvider.cs

```diff
@@ -300,7 +300,7 @@ private async Task<TextDocumentEdit[]> RemapVersionedDocumentEditsAsync(TextDocu
                 FormattingOptions = formattingOptions
             };
 
-            var mappingResult = await _requestInvoker.CustomRequestServerAsync<RazorMapToDocumentEditsParams, RazorMapToDocumentEditsResponse>(
+            var mappingResult = await _requestInvoker.ReinvokeRequestOnServerAsync<RazorMapToDocumentEditsParams, RazorMapToDocumentEditsResponse>(
```

Comment:
Just correcting a mistake during a previous rebase.

Reply:
`CustomRequestServerAsync` was calling SynchronizedRequestAsync which should no longer be used and is going to be removed.